src/Core/Form/IdentifiableObject/DataHandler/CategoryFormDataHandler.php

```diff
@@ -104,14 +104,17 @@ public function update($categoryId, array $data)
         $command = $this->createEditCategoryCommand($categoryId, $data);
 
         $this->commandBus->handle($command);
-
```

Comment:
You can only add a certain amount of menuThumbnailImages. If you are at that amount warning will be displayed that you are at maximum. But you can still add another file.

So I thought it would make sense to disable form input as well since you shouldn't be adding more images anyway(and if you delete one the images page will be reloaded). 
But if I disable input it means menu_thumbnail_images won't be passed to CategoryFormDataHandler anymore, thus I add default value here. 

I tried to do it with readonly instead of disabled but it didn't disable the file input

Reply:
@JevgenijVisockij you can hack it like this:

```
input[type=file][readonly] {
    pointer-events: none;
}
```

but still available if you tab into it, if you still have this issue we can ask @NeOMakinG for an advice